packages/icons/src/library/columns.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * WordPress dependencies
+ */
+import { G, Path, SVG } from '@wordpress/primitives';
+
+const columns = (
+	<SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+		<G>
```

Comment:
The `<G>` here means "group", which makes it grouped when opened in Illustrator. It can also make sense when you create complex SVG illustrations, where you might want to apply a CSS class to a group of sub-elements. 

But for icons, they should probably be as flat as possible, and the `<G>`'s can be removed.